### PR TITLE
[NEW] Omnichannel inquiry queue

### LINKED ISSUE #2233: rocketchat app ios or android does not appear livechat channels
I am using the latest version of the app I have already tested the Iphone and Android, the internal channels work perfectly but the channels created in livechat do not appear

### PR DESCRIPTION
**Pull request by djorkaeffalexandre**
## Proposed changes

Now you'll be able to see omnichannel queue also on mobile client.

## Issue(s)	

Closes #2233.

## How to test or reproduce

You should enable showQueue on your routingConfig of omnichannel, using Manual Selection.
`Administration > Omnichannel > Routing > Omnichannel Routing Method > Manual Selection`

## Screenshots
Screen Shot 2020-07-30 at 11 14 29
Screen Shot 2020-07-30 at 11 14 34
Screen Shot 2020-07-30 at 11 14 38

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [X] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [X] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/i18n/locales/en.js [hunk 1]
```diff
@@ -621,5 +622,7 @@ export default {
 	Passcode_app_locked_title: 'App locked',
 	Passcode_app_locked_subtitle: 'Try again in {{timeLeft}} seconds',
 	After_seconds_set_by_admin: 'After {{seconds}} seconds (set by admin)',
-	Dont_activate: 'Don\'t activate now'
+	Dont_activate: 'Don\'t activate now',
+	Queued_chats: 'Queued chats',
+	Doesnt_have_any_chat_on_the_queue: 'Doesn\'t have any chat on the queue'
```
Comment (diegolmello) [inline app/i18n/locales/en.js:627]: "Queue is empty"

File: app/i18n/locales/en.js [hunk 2]
```diff
@@ -621,5 +622,7 @@ export default {
 	Passcode_app_locked_title: 'App locked',
 	Passcode_app_locked_subtitle: 'Try again in {{timeLeft}} seconds',
 	After_seconds_set_by_admin: 'After {{seconds}} seconds (set by admin)',
-	Dont_activate: 'Don\'t activate now'
+	Dont_activate: 'Don\'t activate now',
```
Comment (diegolmello) [inline app/i18n/locales/en.js:625]: Missing pt-BR

File: app/views/QueueListView.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+		const { queued, theme } = this.props;
+		return (
+			<SafeAreaView testID='queue-list-view' theme={theme} style={{ backgroundColor: themes[theme].backgroundColor }}>
+				<StatusBar theme={theme} />
+				<FlatList
+					ref={this.getScrollRef}
+					data={queued}
+					extraData={queued}
+					keyExtractor={keyExtractor}
+					style={[{ backgroundColor: themes[theme].backgroundColor }]}
```
Comment (diegolmello) [inline app/views/QueueListView.js:135]: Unnecessary array

File: app/views/RoomsListView/index.js [hunk 4]
```diff
@@ -671,6 +676,19 @@ class RoomsListView extends React.Component {
 		}
 	};
 
+	goQueue = () => {
```
Comment (diegolmello) [inline app/views/RoomsListView/index.js:679]: Add analytics

File: app/views/RoomsListView/index.js [hunk 5]
```diff
@@ -960,7 +981,9 @@ const mapStateToProps = state => ({
 	useRealName: state.settings.UI_Use_Real_Name,
 	appState: state.app.ready && state.app.foreground ? 'foreground' : 'background',
 	StoreLastMessage: state.settings.Store_Last_Message,
-	rooms: state.room.rooms
+	rooms: state.room.rooms,
+	queueSize: state.inquiry.queued.length,
```
Comment (diegolmello) [inline app/views/RoomsListView/index.js:985]: I'm thinking if we need a selector for this.
WDYT?

Comment (djorkaeffalexandre) [reply app/views/RoomsListView/index.js:985]: I agree that is a good change

File: app/lib/rocketchat.js [hunk 6]
```diff
@@ -826,6 +841,16 @@ const RocketChat = {
 		// RC 0.26.0
 		return this.methodCallWrapper('livechat:changeLivechatStatus');
 	},
+	getInquiriesQueued() {
+		// RC 2.4.0
+		return this.sdk.get('livechat/inquiries.queued');
+	},
+	takeInquiry(inquiryId) {
+		// this inquiry is added to the db by the subscriptions stream
```
[the remark this answers is not in the document]
Comment (djorkaeffalexandre) [reply app/lib/rocketchat.js:849]: it'll be added to database as a sub/room and removed from the queue on redux

**Pull request merged**